views/containers/Character.jsx

```diff
@@ -0,0 +1,68 @@
+import { connect } from 'react-redux';
+import React from 'react';
+import { characterMove } from '../actions/Character';
+import keyEmitter from '../KeyEventEmitter';
+
+const ReactTHREE = require('react-three');
+const THREE = require('three');
+const Object3D = ReactTHREE.Object3D;
+const Mesh = ReactTHREE.Mesh;
+
+const keys = ['W','A','S','D'];
+
+class MovingCharacter extends React.Component {
+  componentDidMount = () => {
+      this.props.onMount()
+  };
+
+  componentWillUnmount = () => {
+      this.props.onUnmount()
+  };
+
+  render() {
+    const material = new THREE.MeshBasicMaterial({
+      color: 0xff0000,
+    });
+    const headGeometry = new THREE.SphereGeometry(5, 32, 32);
+    const bodyGeometry = new THREE.CylinderGeometry( 6, 6, 10, 32 );
+    const legGeometry = new THREE.CylinderGeometry( 3, 3, 10, 32 );
+    const headPosition = new THREE.Vector3(0,25,0).add(this.props.position);
+    const bodyPosition = new THREE.Vector3(0,15,0).add(this.props.position);
+    const legPosition = new THREE.Vector3(0,5,0).add(this.props.position);
+    return (
+      <Object3D>
+        <Mesh position={headPosition} geometry={headGeometry} material={material} />
+        <Mesh position={bodyPosition} geometry={bodyGeometry} material={material} />
+        <Mesh position={legPosition} geometry={legGeometry} material={material} />
+      </Object3D>
```

Comment:
^^ This should be a dumb component, so move the visual part of it and contain it here.